src/controllers/factorycontroller.js

```diff
@@ -44,17 +44,10 @@ class FactoryController extends OrderController
     factoryOrder(formValues)
     {
         var products = OrderController._makeOrder(formValues);
-        var orderCost = products.reduce((sum, prod) => sum + prod.value(), 0);
-
-        if (this.validateOrder(products)) {
-            this._updateMoney(-orderCost);
+        var order = new Order(products, GAME.model.config.orderTransportDuration, GAME.model.orders.length);
```

Comment:
There's no need to push globals into a function. Just have those variables as the default arguments in the constructor for Order.